packages/ketcher-polymer-editor-react/src/components/modal/monomerConnection/MonomerConnections.tsx

```diff
@@ -19,6 +19,8 @@ import {
 } from './styledComponents';
 import { MonomerConnectionProps } from '../modalContainer/types';
 
+type LeavingGroupValue = 'O' | 'OH' | 'H';
```

Comment:
If anyone has a feeling that this declaration would look better in another file (some place where all typings live?) please let me know. I just don't know where I should put it.